src/openrct2/scripting/bindings/game/ScContext.hpp

```diff
@@ -45,6 +45,36 @@ namespace OpenRCT2::Scripting
         }
 
     private:
+        std::vector<DukValue> allPlugins_get()
+        {
+            // Get all of the plugins from the script engine
+            OpenRCT2::Scripting::ScriptEngine& scriptEngine = GetContext()->GetScriptEngine();
+            std::vector<std::shared_ptr<OpenRCT2::Scripting::Plugin>> allPlugins = scriptEngine.GetPlugins();
```

Comment:
```suggestion
            const auto& allPlugins = scriptEngine.GetPlugins();
```
Making a copy of this is quite expensive especially with the shared_ptrs being the elements.

Reply:
Thanks for this little explanation, helped a lot with my understanding of references